STACpopulator/extensions/file.py

```diff
@@ -0,0 +1,84 @@
+"""FileHelper module."""
+
+import functools
+import logging
+from typing import Dict, Optional, TypeVar
+
+import pystac
+import requests
+from pydantic import ConfigDict, Field
+from pystac.extensions.file import FileExtension
+from requests import Session
+
+from STACpopulator.extensions.base import ExtensionHelper
+
+# Constants
+T = TypeVar("T", pystac.Asset, pystac.Link)
+logger = logging.getLogger(__name__)
+
+
+class FileHelper(ExtensionHelper):
+    """Helper to handle file info from elements of types Asset and Link."""
+
+    access_urls: Dict[str, str]
+    asset_key: str = "HTTPServer"
+    session: Optional[Session] = Field(default=None, exclude=True)
```

Comment:
Those are typically annotated with [`PrivateAttr`](https://docs.pydantic.dev/latest/api/fields/#pydantic.fields.PrivateAttr) to be explicit about it, but I am not sure if using it interferes with the  `__init__` call (as mentioned in the doc)?